src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -478,6 +478,11 @@ struct VARIABLES_DUALES_INTERCONNEXIONS
 
 struct PROBLEME_HEBDO
 {
+    PROBLEME_HEBDO() = default;
+    PROBLEME_HEBDO(const PROBLEME_HEBDO&) = delete;
+    PROBLEME_HEBDO(PROBLEME_HEBDO&&) = default;
+    PROBLEME_HEBDO& operator=(PROBLEME_HEBDO&& other) = default;
```

Comment:
See "[rule of 5](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-copy-move-or-destructor-function-define-or-delete-them-all)": let's define also explicitly the = operator as deleted, for clarity sake.